src/plugins/data/README.md

```diff
@@ -111,7 +111,7 @@ The `SearchSource` API is a convenient way to construct and run an OpenSearch se
 
 #### Default Search Strategy
 
-One benefit of using the low-level search API, is partial response support in X-Pack, allowing for a better and more responsive user experience.
+One benefit of using the low-level search API, is allowing for a better and more responsive user experience.
```

Comment:
I'm not sure if this is true - I may make a follow-up issue, but for now, we can make it a little more readable.

```suggestion
One benefit of using the low-level search API is that it allows for a better and more responsive user experience with partial responses.
```

Reply:
@joshuarrrr Great. Will make those changes and append to this PR. 👍